joomla/joomla-cms
# [4.0] Missing language strings

### Summary of Changes
Add of missing language strings


### Testing Instructions
1. Try to install a manifests with a api section but missing api files. If you like, you can take this zip:
[com_foos.zip](https://github.com/joomla/joomla-cms/files/4407168/com_foos.zip)
2. You see this message
![Extensions  Install   test   Administration](https://user-images.githubusercontent.com/9974686/77994521-76f7e680-732a-11ea-910c-05b259f3af33.png)
3. Apply this patch and redo the installation with the same manifest.
You see the same message, but the language string is translated.
![Extensions  Install   test   Administration(1)](https://user-images.githubusercontent.com/9974686/77994588-942cb500-732a-11ea-95b2-9bf779b55383.png)





### Expected result



### Actual result



### Documentation Changes Required
